deeppavlov/models/bert/bert_classifier.py

```diff
@@ -0,0 +1,253 @@
+        self.num_warmup_steps = num_warmup_steps
+        self.weight_decay_rate = weight_decay_rate
+
+        if self.multilabel and not self.one_hot_labels:
+            raise RuntimeError('Use one-hot encoded labels for multilabel classification!')
+
+        if self.multilabel and not self.return_probas:
+            raise RuntimeError('Set return_probas to True for multilabel classification!')
+
+        self.bert_config = BertConfig.from_json_file(str(expand_path(bert_config_file)))
+
+        if attention_probs_keep_prob is not None:
+            self.bert_config.attention_probs_dropout_prob = 1.0 - attention_probs_keep_prob
+        if hidden_keep_prob is not None:
+            self.bert_config.hidden_dropout_prob = 1.0 - hidden_keep_prob
+
+        self.sess_config = tf.ConfigProto(allow_soft_placement=True)
+        self.sess_config.gpu_options.allow_growth = True
+        self.sess = tf.Session(config=self.sess_config)
+
+        self._init_graph()
+
+        self._init_optimizer()
+
+        self.sess.run(tf.global_variables_initializer())
+
+        if pretrained_bert is not None:
+            pretrained_bert = str(expand_path(pretrained_bert))
+
+        if tf.train.checkpoint_exists(pretrained_bert) \
+                and not tf.train.checkpoint_exists(str(self.load_path.resolve())):
+            logger.info('[initializing model with Bert from {}]'.format(pretrained_bert))
+            # Exclude optimizer and classification variables from saved variables
+            var_list = self._get_saveable_variables(
+                exclude_scopes=('Optimizer', 'learning_rate', 'momentum', 'classification'))
+            saver = tf.train.Saver(var_list)
+            saver.restore(self.sess, pretrained_bert)
+
+        if self.load_path is not None:
+            self.load()
+
+    def _init_graph(self):
+        self._init_placeholders()
+
+        self.bert = BertModel(config=self.bert_config,
+                              is_training=self.is_train_ph,
+                              input_ids=self.input_ids_ph,
+                              input_mask=self.input_masks_ph,
+                              token_type_ids=self.token_types_ph,
+                              use_one_hot_embeddings=False,
+                              )
+
+        output_layer = self.bert.get_pooled_output()
+        hidden_size = output_layer.shape[-1].value
+
+        with tf.variable_scope('classification'):
+            output_weights = tf.get_variable(
+                "output_weights", [self.n_classes, hidden_size],
+                initializer=tf.truncated_normal_initializer(stddev=0.02))
+
+            output_bias = tf.get_variable(
+                "output_bias", [self.n_classes], initializer=tf.zeros_initializer())
+
+        with tf.variable_scope("loss"):
+            output_layer = tf.nn.dropout(output_layer, keep_prob=self.keep_prob_ph)
+            logits = tf.matmul(output_layer, output_weights, transpose_b=True)
```

Comment:
Why not `tf.layers.dense`?

Reply:
to make sure compatibility with BERT classifier from official Google repo